src/components/App.jsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react'
+import React from "react";
+import ContactsApp from "./ContactsApp";
 
-const App = ({ children }) => (
-  <div className='app-wrapper o-layout--2panes'>
-    <main className='app-content'>
-      <div>Hello World!</div>
+const App = () => (
+  <div className="app-wrapper o-layout--2panes">
```

Comment:
`o-layout--2panes` seems wrong? What does the `o` prefix means? And we don't have 2 panes.